exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -70,6 +70,11 @@
 
   private static final Logger logger = LogManager.getLogger(QaService.class);
 
+  private static int CREATE_COUNTER_TX_ID = 0;
```

Comment:
The constants duplicate the values in QaTransaction enum — shall probably re-use (or one of them shall be deleted)?

Reply:
Yes, was going to delete the enum